docs/src/references/certificate-management.md

```diff
@@ -0,0 +1,358 @@
+:::
+
+## Certificate renewal
+
+Once the %%te%% device is connected to Cumulocity, its certificate can be renewed by using the existing connection to Cumulocity. If the device is using SystemD as its init system, then the certificate certificate renewal process will be automatic, and it will be renewed before the certificate expires (the default is 30 days before expiration, but the value can be configured via the tedge configuration).
+
+The Cumulocity certificate-authority feature provides a new API endpoint for devices to request a new certificate provided the device can still communicate with the tenant it is requesting a certificate from. The certificate renewal is authenticated by way of a JWT which is requested over the MQTT connection using the current certificate. If for any reason the certificate expires, and the device can no longer communicate with Cumulocity, then the device will need to be registered again.
+
+To ensure a smooth transition to the new certificate, the certificate renewal process is done in two distinct steps as described below:
+
+1. Request a new certificate authenticating with the current certificate. The newly issued certificate is downloaded to the device, however it will not overwrite the current certificate. Instead the newly issued certificate will be saved next next to the current certificate and be given the `.new` filename suffix.
+2. Verify the connection to Cumulocity using the new certificate candidate, and if the connection to Cumulocity is successful, then the new certificate candidate will overwrite the previous certificate. If the connection is not successful, then the existing certificate is not touched, and the certificate candidate will also remain untouched.
+
+For additional options on how the certificate registration and renewal can be configured (e.g. how many days before the certificate expires should the certificate be renewed), check out the [Configuration](#configuration) section.
+
+### Automated certificate renewal (systemd)
+
+If you're using a device with SystemD, connected to Cumulocity and have the , then the certificate renewal service will be installed, and it will take care of renewing the device certificate automatically.
+
+The automated certificate renewal is implemented using SystemD and the components and their role are listed below:
+
+|SystemD Unit|Description|
+|------------|-----------|
+|tedge-cert-renewer@c8y.timer|Timer which runs periodically to trigger the `tedge-cert-renewer@c8y` service. By default it runs every hour on the hour (with built-in jitter).|
+|tedge-cert-renewer@c8y.service|Service which performance the renewal if the certificate is about to expire as defined by in the %%te%% configuration|
```

Comment:
```suggestion
|tedge-cert-renewer@c8y.service|Service which performs the renewal if the certificate is about to expire as defined in the %%te%% configuration|
```

Reply:
Fixed in f3a099ff38ed6949b893fdb6e704fa013c82df16